### ISSUE TITLE
Inline suppression comments

### ISSUE THREAD

**Issue by languitar** [amperser/proselint#708]
Are there any special markers (like comments in latex) that can be used to turn of linting for a certain line? If not, such a thing would be very nice to avoid some false positives.

**Comment by fphammerle** [comment]
example:
```tex
resolution \SI{0.2x0.2x0.2}{\mm}
```
reports
> typography.symbols.multiplication_symbol Use the multiplication symbol ×, not the letter x.

However, the use of `×` would introduce a syntax error. `x` is fine here.
